### ISSUE TITLE
question: ie 11 compatibility

### ISSUE THREAD

**Issue by semoal** [accessible-ui/popover#2]
Hi mate, this component works amazing.

Could we create a version of the library with ie 11 compatibility out of the box?

I've tried to compile all this libraries with a babel.config.js with IE 11 on target, but crashes...

> "react-portalize",
  "@accessible/popover",
  "@accessible/button",
  "@accessible/tabbable",
  "@accessible/use-conditional-focus",
  "@accessible/use-id",
  "@accessible/use-keycode",
  "@react-hook/debounce",
  "@react-hook/event",
  "@react-hook/latest",
  "@react-hook/merged-ref",
  "@react-hook/passive-layout-effect",
  "@react-hook/switch",
  "@react-hook/throttle",
  "@react-hook/window-scroll",
  "@react-hook/window-size",

```
The error: TypeError: Assignment to read-only properties is not allowed in strict mode
in Target 
```

**Comment by jaredLunde** [comment]
Eh, this is not something that is a responsibility of the library. It's in the user's hands to target the browsers they need to support. Even if I were to provide an IE11-compatible output, that user is one `Object.entries` away from breaking IE11 in some other library. Non-standard features, sure, those should be covered by the library. But not anything else. It only serves to bloat the JS output for those of us who unabashedly denounce IE11 and refuse to support it further. The error you're displaying here only hardens my resolve.

**tldr; compile your `node_modules`**
